css/styles.css

```
@@ -1 +1,35 @@
-/* CSS */
\ No newline at end of file
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Roboto', sans-serif;
```

_Roboto_ is not default (build-in OS) font, so we need to import it from somewhere.
There are few ways to load custom fonts:
- load from local files
- load from external resources (e.g.: _Google Fonts_)

Another thing to consider here is that if user doesn't have exact font loaded
it will take next on the list to load. In that case, if we don't have _Roboto_,
the sans-serif will be loaded. We can add here _Helvetica_ and _Arial_ as fallback:
```suggestion
    font-family: 'Roboto', Helvetica, Arial sans-serif;
```

Please check these resources how you can load fonts:
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link
- https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face